pulp_container/app/registry_api.py

```diff
@@ -328,11 +302,9 @@ def get_dr_push(self, request, path, create=False):
             elif create:
                 try:
                     with transaction.atomic():
-                        repo_serializer = serializers.ContainerPushRepositorySerializer(
-                            data={"name": path}, context={"request": request}
+                        repository = serializers.ContainerPushRepositorySerializer.get_or_create(
+                            {"name": path}
                         )
-                        repo_serializer.is_valid(raise_exception=True)
-                        repository = repo_serializer.create(repo_serializer.validated_data)
                         distribution.repository = repository
                         distribution.save()
                 except IntegrityError:
```

Comment:
this can be removed

Reply:
I was unsure if `save()` could raise any error. By the way, should we enforce saving the object just in time with `force_update=True`?

Reply:
i don't think so https://docs.djangoproject.com/en/dev/ref/models/instances/#ref-models-force-insert